dataprep/eda/basic/render.py

```diff
@@ -89,6 +89,52 @@ def _make_title(grp_cnt_stats: Dict[str, int], x: str, y: str) -> str:
     return f"{y} by {x}"
 
 
+def _format_xaxis(fig: Figure, minv: int, maxv: int) -> None:
+    """
+    Format the x axis for histograms
+    """
+    # interval
+    gap = (maxv - minv) / 4.5
```

Comment:
Is the 4.5 related to the default bin number 5? Please make a comment about it and it is better to use a variable to store the default value (e.g., line 108).